propn-ner-comparator/README.md

```diff
@@ -0,0 +1,19 @@
+# PROPN vs NER tag comparator
+
+A quick script that compares the coverage of tokens tagged with PROPN in
+Universal Dependencies with NER tags as part of the UniversalNER project.
+
+## Installation
+
+The dependencies are managed via [Poetry](https://python-poetry.org/docs/).
```

Comment:
I hope using Poetry isn't too much of a hassle -- if that ends up being the case, I'd be happy to simply dump `requirements.txt`.

Reply:
I'd prefer if we kept the package dependencies as simple as possible, would you mind dumping a `requirements.txt`?

Reply:
Roger that, I will dump them too.